Project1

sergiocortessat opened the pull request:
First, pull to request a review. The HTML Skeleton is missing the footer part as we couldn't complete it, as well is missing the boxes of suggestions and picks. As well is missing Styling for image and video sections and the bottom sections. Unfortunately, we couldn't finish the complete thing and we must continue with the missing parts still. Thank you

Commit:
Initial push of README, Styles, Icons and Images

Commit:
Add HTML Skeleton

Commit:
Add of Linters check, gitignore, .github

Commit:
Modification clases CSS, HTML file.

Commit:
NAVBAR HTML CSS

Commit:
styling headers

Commit:
add more HTML structure

Commit:
changes <article> for <div>

Commit:
changes to NAV, MAIN

Commit:
Changes further to NAV. Added flex

mahtsham committed:
add nav buttons styling

mahtsham committed:
add new color and font-family

mahtsham committed:
add hover

Commit:
add last changes to NAV-Zindex 1

andresporras3423 reviewed (changes requested):
### Status: Changes Required :recycle:
**Hello Ahtsham Ahtsham [status: Enrolled] & Sergio Cortes Satizabal** 
Good job with your first submission for this project :muscle:  :muscle:  
It needs some improvements before you can go ahead. Remember, the general structure and general appearance should be like the original. I will show you just some of the mismatches:

- [ ] The README file should include a screenshot of the Live Demo
- [ ] The README file should include the link to the Live Demo
- [ ] The theory of inflation section doesn't match the original:

| Yours | Original |
|:---:|:---:|
|![image](https://i.ibb.co/QbN5V4s/review14.png) | ![image](https://i.ibb.co/z2rXMDJ/review15.png) |

- [ ] The footer is missing (or it doesn't match the original):

| Yours | Original |
|:---:|:---:|
|![image](https://i.ibb.co/DzpzJDP/review16.png) | ![image](https://i.ibb.co/GVTwD4H/review17.png) |

- [ ] The editor's pick section doesn't math the original:

| Yours | Original |
|:---:|:---:|
|![image](https://i.ibb.co/jJh9RLd/review18.png) | ![image](https://i.ibb.co/rm5ZhF6/review19.png) |

- [ ] Remove commented code from your index.html
- [ ] Elements should be aligned with a combination of all three techniques (float, flex and grid)
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

Commit:
add aside block images

Commit:
add style to footers, comments button, aside positioning and boxing

Commit:
add comments, organized footers.

mahtsham committed:
The inflation part

Commit:
styling HTML

Commit:
Merge branch 'project1' of github.com:sergiocortessat/Week1_Positioning-and-Floating-Elements- into project1

Commit:
eliminate bad code

Commit:
minor changes

sergiocortessat committed:
Update README.md

sergiocortessat committed:
Update README.md

sergiocortessat committed:
Update README.md

Commit:
image for read.me

Commit:
Merge branch 'project1' of github.com:sergiocortessat/Week1_Positioning-and-Floating-Elements- into project1

sergiocortessat committed:
Adding prper screenshot

Commit:
adding image to read.me

Commit:
Merge branch 'project1' of github.com:sergiocortessat/Week1_Positioning-and-Floating-Elements- into project1

sergiocortessat committed:
Update README.md

sergiocortessat committed:
Update README.md

Commit:
fixed alineation bottom

Commit:
Merge branch 'project1' of github.com:sergiocortessat/Week1_Positioning-and-Floating-Elements- into project1

Commit:
Fixed linters

Commit:
Fixed linters

Commit:
added file to gitignore.didnt pass linter

Commit:
added file to gitignore.didnt pass linter

Commit:
added file to gitignore.didnt pass linter

sergiocortessat commented:
New request to review. We have added the missing parts and given positioning to the page using some GRID, position, and Flex. As well we have added the required Readme file with description, live demo, and screenshot. The middle section called Inflation theory was designed with GRID but misses the background fixed background image @mahtsham @andresporras3423

mahtsham commented:
Hey Sergio,
I am working on that inflation issues. and also I will try to setup the
bottom part. We will submit the whole complete project tomorrow before
lunch time with no bugs, if they tell us to modify. We have a hard deadline
for tommorrow.

On Thu, Jan 14, 2021, 21:55 Sergio Cortes Satizabal <
notifications@github.com> wrote:

> New request to review. We have added the missing parts and given
> positioning to the page using some GRID, position, and Flex. As well we
> have added the required Readme file with description, live demo, and
> screenshot. The middle section called Inflation theory was designed with
> GRID but misses the background fixed background image @mahtsham
> <https://github.com/mahtsham> @andresporras3423
> <https://github.com/andresporras3423>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sergiocortessat/Week1_Positioning-and-Floating-Elements-/pull/1#issuecomment-760324664>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQOHGDU4NXJGE6PTUJKMEKDSZ4OZPANCNFSM4WBCKZYA>
> .
>

layzrd reviewed (changes requested):
### Status: Require changes 🌳 ♲
Hi @sergiocortessat  and @mahtsham  , I am Abdulaziz Redi Ali, your TSE for this PR.  👏👏  I appreciate your hard work in replicating nytimes.com.   Kindly fix the following issues.

[REQUIRED]

- [ ] Fix the left-most part of the navigation bar to have a similar appearance as the original page
- [ ] Fix the images on the "Theory of inflation" section  by giving source to the "img" tags
- [ ] Add the "Most populars" section and the "suggested newsletters" section on the bottom part of the page as on the original site
- [ ] The footer should be visually similar to the original page. 

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

Commit:
fixed footer, header

Commit:
fixed left-navbar icons

mahtsham committed:
add images

mahtsham committed:
add images and style to inflation

Commit:
last fixes of style

Commit:
last fixes of style

Commit:
final style before check

sergiocortessat commented:
Pull request to check las changes requested. Top-left navbar alignment, Theory of inflation GRID, Suggestions GRID, Overall Styling, Fix bottom footer to match original @mahtsham

sergiocortessat committed:
Update README.md

sergiocortessat commented:
Added license missing on Read.me

ebeagusamuel reviewed (changes requested):
# Changes Requested :recycle: 
### Hey there :wave:
#### You are almost done with this project. so a minor issue to fix.
- [ ] Please update the footer section to look like the original, juts as required by the previous TSE.
- [ ] Please update the most popular section of your cloned page. It is too aligned to the right and that is not how it is on the original page.
#### Do request another review when you are done.
slack@ebeagusamuel)

Commit:
Add missin html footer

Commit:
Add style editors and footer

Commit:
Merge branch 'project1' of github.com:sergiocortessat/Week1_Positioning-and-Floating-Elements- into project1

Commit:
erased commented file

Commit:
Add linters checked

Commit:
Add linters for webhint

Commit:
Add linters for webhint

Commit:
modified size of html

Commit:
set margin to 600px

sergiocortessat commented:
Added last requests to change. Footer matching more the original one and editors pick set further from the right margin @mahtsham

Temmarie reviewed (changes requested):
## Changes Required! :recycle:

### Hi @sergiocortessat and @mahtsham   ! :wave: :wave:

#### You have done well so far :clap: However, there are changes required for you to make in order for this project to be approved. Kindly work on them and submit them for another code review.

**General**
- [ ] For future references, use [font awesome](https://fontawesome.com/v4.7.0/icons/) for the icons on the page, especially the footer.
- [ ] Use [this](https://chrome.google.com/webstore/detail/font-picker/mmjbimgpcbaegjiieojddickpjbdkeej) chrome extension to determine the font family and sizes used in the actual site.
- [ ] Use [this](https://chrome.google.com/webstore/detail/svg-export/naeaaedieihlkmdajjefioajbbdbdjgp?hl=en-GB) SVG picker for SVG images for future reference.


**Recommended**

- [ ] The Most popular section is still tilted to the right, please fix this.
- [ ] The "big telescope" image is really big compared to that on the actual site, please adjust the size.
- [ ] The editor's pick sidebar needs more work. Compare the differences.

**Original**

![Screenshot (13)](https://user-images.githubusercontent.com/49621279/104803410-f190d600-5785-11eb-950e-567cacc17d00.png)

**Your clone**

![Screenshot (14)](https://user-images.githubusercontent.com/49621279/104804674-50eee600-5786-11eb-876f-16027bb3463d.png)

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

mahtsham committed:
add style to telescope picture

mahtsham committed:
add work on editor's pick

mahtsham commented:
Hello TSE,
We have made all the changes that come from your side. We have learned a lot and your reviews help us to build our concepts.
In this project, we have changed all the required changes which were needed, and it's working well on our side. 
So, if you still notice that any of the parts is more towards the right side then check it on 1440px resolution. As we have not learned about the responsiveness of the page, I think that screen size is messing up on your side.

Commit:
Suggestions,Footer fixed to match

Commit:
add margin bottom-homepage

layzrd reviewed (approved):
### Status: Approved 🙌 🎇
Hi @sergiocortessat and @mahtsham, I am  [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  Congratulations 🎊 👏, you did very well on this project.

Now, you can move ahead.

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)
